jenkins/cinder-sofs-validate/50-run.sh

```diff
@@ -4,8 +4,7 @@ sudo groupadd jenkins
 sudo gpasswd -a jenkins jenkins
 sudo chown -R jenkins:jenkins /opt/git/
 
-sudo useradd -m -U stack
-sudo gpasswd -a stack wheel
+echo "#includedir /etc/sudoers.d" | sudo tee -a /etc/sudoers
```

Comment:
I was a bit confused by the `#`, expecting it to be a comment (which it is, except when followed by `include` and some other things according to `man 5 sudoers`).